bounties/BA122401.md

```diff
@@ -0,0 +1,41 @@
+---
+code: BA122401
+title: Rocket Pool Support Payments
+proposal-author: epineph
+creation-date: 2024-05-28
+external-status: open
+summary: Support users within Rocket Pool's Discord support channel.
+reward: 
+  unit: USD
+  max: 54000
+  min: 54000
+repeatable: true
+expiry: August 31, 2024
+application-url: https://dao.rocketpool.net/t/round-12-call-for-bounty-applications-deadline-is-may-7/2919/2?u=shfryn
+skillsets:
+  - development
+  - protocol-knowledge
+  - node-operation
+contacts:
+  - ShfRyn
+  - epineph
+---
+
+## Summary 
+Provide support for questions posed by Rocket Pool community members in the # support discord channel; questions come largely from Node Operators and those hoping to be Node Operators. 
+
+## Verification
+
+Provide user support in the # support channel of discord, under the following terms:
+
+1) Provide at least 1 hour of work providing #support discord channel in EACH of the three months June-August 2024, as measured by RocketScrape; *alternatively provide a total 15 hours TOTAL over those 3 months.
```

Comment:
Part of milestone

Reply:
Like... Assume someone that has no context of how rocket pool support works. What are they actually supposed to do here? Like, you're putting it on a portal whose audience is someone is familiar with crypto but may not be familiar with RP.

Reply:
It's part of milestone and verification?